spec/template.dd

```diff
@@ -1394,11 +1394,27 @@ $(GNAME Constraint):
             ...
         }
         ...
-        foo!(3)();  // ok, matches
-        foo!(4)();  // error, no match
+        foo!(3)();  // OK, matches
+        foo!(4)();  // Error, no match
+        ---
+    )
+    
+    $(P Template constraints can be used with aggregate types (structs, classes, unions). 
+        Constraints are effectively used with library module "std.traits":
+    
+        ---
+        import std.traits;
+        
+        struct Bar(T)
+            if( isIntegral!T )
```

Comment:
Can you be more specific? 
This seems to abide, according to "https://dlang.org/dstyle.html". Further, I just copied the style from the example immediately above.